src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -62,7 +61,7 @@ See the [Amazon Cognito documentation]( https://docs.aws.amazon.com/cognito/late
 Once you have chosen your management method you can configure your Auth category and install the library to use with your application.
 
 During this workflow you can set up your password formats. If you have not worked with passwords formats recently we recommend taking a closer look at the best practices before you continue.
-
```

Comment:
The expander that follows did not have a clear transition with the other content so I added a bit to this. However, it looks like from the switchers that only Studio requires password decisions this early do we want to move this information to that switcher specifically? If we do that I recommend we take it out of the expander.

Reply:
hmmm - that is correct - you can customize your password formats though at a later time if needed. So it is worth having it here. A developer can change it anytime as well.